source/stylesheets/components/_case_study.scss

```diff
@@ -0,0 +1,125 @@
+// CaseStudy
+//
+// A compoent to showcase a project
+//
+// .CaseStudy--articles - Uses oncostats' background color
+// .CaseStudy--oncostats - Users articles' background color
+// .CaseStudy--left - Puts the image on the left
+//
+// markup:
+// <div class="CaseStudy {$modifiers}">
+//   <img class="CaseStudy-image">
+//   <div class="CaseStudy-details">
+//     <div class="CaseStudy-fixedBackground"></div>
+//     <h2 class="CaseStudy-title">Some title</h2>
+//     <p>Some html</p>
+//     <p>More HTML</p>
+//   </div>
+// </div>
+//
+// Styleguide 5.22
+
+$CaseStudy-colors: (articles, #cdb187), (oncostats, #6dd4e0);
+$CaseStudy-shadow: 0 20px 60px 0 rgba(#000, 0.25);
+
+.u-overflowHidden {
+  overflow: hidden;
+}
+
+.CaseStudy {
+  position: relative;
+
+  display: flex;
+  flex-direction: column;
+
+  max-width: $Theme-grid-maxWidth;
+  margin: 0 auto;
+
+  @include media('>smallDesktop') {
```

Comment:
I see what you mean; we could define a set of acceptable numbers for the media queries and use them instead of this alias. But I think that naming still helps to know what screen size we are targeting; maybe there is a naming system that isn't coupled to the screen type?